tests/README.md

```diff
@@ -110,6 +110,7 @@ s3://<bucket-url>/microvm-images/
     <microvm_test_image_folder_n>/
```

Comment:
Since you're touching/fixing this file, please also update the `S3` bucket name from `spec.firecracker` to `spec.ccfc.min`, also the `tree` has changed from:

> `s3://<bucket-url>/microvm-images/`

to

`s3://<bucket-name>/img/`

Reply:
Yep, updated.